fsw/linux/libcan.c

```diff
@@ -29,12 +29,12 @@ int32_t can_init_dev(can_info_t* device)
   retVal = can_set_bitrate(device->handle, device->bitrate);
   if (retVal < 0) 
```

Comment:
@mgrubb-gsfc - The CW_addr logic in the libcan.h and sim can be removed

Reply:
Opened new issue to capture this in since it doesn't imped anything for this merge request:
https://github.com/nasa/nos3/issues/110